mvc/src/main/java/nextstep/mvc/controller/tobe/AnnotationHandlerMapping.java

```diff
@@ -20,11 +30,50 @@ public AnnotationHandlerMapping(Object... basePackage) {
         this.handlerExecutions = new HashMap<>();
     }
 
+    @Override
     public void initialize() {
+        Reflections reflections = new Reflections(basePackage);
+
+        Map<String, ? extends Class<?>> controllers = reflections
+            .getTypesAnnotatedWith(Controller.class).stream()
+            .collect(toMap(Class::getName, controller -> controller));
+
+        List<Method> requestMappingMethods = controllers.values().stream()
+            .map(Class::getDeclaredMethods)
+            .flatMap(Arrays::stream)
+            .filter(method -> method.getDeclaredAnnotation(RequestMapping.class) != null)
```

Comment:
isAnnotationPresent을 사용해도 되겠네요!
```suggestion
           .filter(method -> method.isAnnotationPresent(RequestMapping.class))
```

Reply:
대박 이런게 있다니 !!